fossunited/chapters/doctype/foss_event_rsvp/foss_event_rsvp.py

```
@@ -75,6 +75,11 @@ def get_context(self, context):
                 "options": "\nStudent\nProfessional\nOther",
                 "reqd": 1,
             },
+            {
+                "fieldname": "subscribe_chapter_mailing",
+                "fieldtype": "Check",
+                "label": f"Yes, I’d like to receive email updates about future events from {self.chapter}.",  # noqa: E501
```

Ruff (0.14.0)

String contains ambiguous `’` (RIGHT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF001)